razorpay/script.js

```diff
@@ -61,7 +61,7 @@ document.addEventListener('DOMContentLoaded', function(event) {
       },
       order_id: defaults.rzp_order_id,
       notes: {
-        prestashop_order_id: defaults.cart_id,
+        prestashop_cart_id: defaults.cart_id,
```

Comment:
This is block tracking again.

Reply:
If the payment flow working without any issue, then there is no issue because we are updating notes again with proper order ID. 
Why I removed, because there may be conflict when we are updating notes with order Id and same cart ID no also present for that.